src/imageio/format/jxl.c

```diff
@@ -0,0 +1,640 @@
+/*
+    This file is part of darktable,
+    Copyright (C) 2021-2022 darktable developers.
+
+    darktable is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    darktable is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with darktable.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "bauhaus/bauhaus.h"
+#include "common/colorspaces.h"
+#include "common/darktable.h"
+#include "common/exif.h"
+#include "common/imageio.h"
+#include "control/conf.h"
+#include "imageio/format/imageio_format_api.h"
+
+#include "jxl/encode.h"
```

Comment:
I suppose here and below we want to use `< >` as those are external OS includes.

Reply:
Done